pkg/ssh/pool.go

```diff
@@ -37,40 +37,68 @@ func startPool(ctx context.Context, serverAddr string, config *ssh.ClientConfig)
 		stopped: false,
 	}
 
-	clientConn, chans, reqs, err := retryNewClientConn(ctx, serverAddr, config, p)
+	var err error
+	var client *ssh.Client
+	t := time.NewTicker(500 * time.Millisecond)
+
+	for i := 0; i < 10; i++ {
```

Comment:
`retryNewClientConn` already does retries

Reply:
yeah, retryNewClientConn validates that you can establish a connection, this part validates that you can send and get back a message from there. I've seen logs where we connect to the SSH but the forwards don't flow, so having this could help us better understand what's going on, and mitigate situations where the sock works, but not the end-to-end connectivity.